### DISCUSSION TITLE
Document docker requirement

### DISCUSSION THREAD

**Discussion by Noel-Jones** [aquasecurity/trivy#6078] [Documentation]
### Description

I have installed according to the installation guide but trivy immediately fails because it cannot download the database. There is no reference to any dependencies on docker being installed, which is implied by the error message telling me that that docker-credential-desktop.exe is not on the path. 

### Suggestions

The install guide needs to make the dependency on docker clear up front and provide at least a link to point people in the right direction.

I appreciate this isn't the correct place to go into the why but:
-  I do not understand why the database requires authentication to download.
- Installing docker is a blocker for me. I am running using Windows Subsystem for Linux and since Docker Desktop is a licenced product I cannot install it. 
- I was directed to Trivy as a replacement to TfSec and I only want to scan a filesystem.

**Comment by knqyf263** [comment]
Trivy doesn't depend on docker.

**Reply by knqyf263** [reply]
And the database doesn't require authentication.
https://aquasecurity.github.io/trivy/v0.49/docs/references/troubleshooting/#denied
